xcell/tests/test_data.py

```diff
@@ -0,0 +1,237 @@
+import glob
+import pytest
+import os
+import yaml
+from xcell.cls.data import Data
+from ..mappers import mapper_from_name
+
+
+def read_yaml_file(file):
+    with open(file) as f:
+        config = yaml.safe_load(f)
+    return config
+
+
+def get_input_file():
+    return './xcell/tests/data/desy1_ebossqso_p18cmbk.yml'
+
+
+def get_config_dict():
+    input_file = get_input_file()
+    config = read_yaml_file(input_file)
+    return config
+
+
+def get_data():
+    input_file = get_input_file()
+    return Data(input_file)
+
+
+def get_data_from_dict():
+    input_dict = get_config_dict()
+    return Data(data=input_dict)
+
+
+def remove_yml_file(config):
+    outdir = config['output']
+    fname = os.path.join(outdir, '*.yml')
+    for f in glob.glob(fname):
+        os.remove(f)
+
+
+def remove_outdir(config):
+    outdir = config['output']
+    if outdir != './xcell/tests/cls/':
+        raise ValueError('Outdir looks odd. Not removing it as precaution')
+    # Using os.rmdir as it will only remove an empty directory
+    elif os.path.isdir(outdir):
+        os.rmdir(outdir)
+
+
+def test_initizalization():
+    input_file = get_input_file()
+    config = read_yaml_file(input_file)
+    with pytest.raises(ValueError):
+        Data(input_file, data=config)
```

Comment:
what is this testing?

Reply:
That you cannot input both a path and a config dictionary

Reply:
ok, great. Can you add a comment for the future?

Reply:
Done